federation/CMakeLists.txt

```diff
@@ -27,6 +28,7 @@ install(
 # Install Headers:
 set(HEADERS_FEDERATION
 ${CMAKE_CURRENT_SOURCE_DIR}/src/endpoint.h
+${CMAKE_CURRENT_SOURCE_DIR}/src/watcher.h
 )
 #   - *.h -> <prefix>/include/${TARGET_NAME}/*.h
```

Comment:
```suggestion
```

Reply:
I think this is a documentation comment, other modules have the same comment